JitsiMeetJS.js

```diff
@@ -335,15 +329,19 @@ export default {
                             newResolution);
 
                         Statistics.analytics.sendEvent(
-                            `${GET_USER_MEDIA_FAIL_}.resolution.${
-                                oldResolution}`);
+                            'warning',
```

Comment:
what determines whether a `createXXXEvent` helper is used or an event is just manually created?

Reply:
Convenience. If I only needed a name, I used a simple constant for the name. If I needed it to set something (e.g. the "source"), then I used a factory function.

Reply:
For someone who doesn't work in lib-jitsi-meet or even does analytics, what would the thought process be when adding a new event and when to use a string vs factory? How would that person know how to proceed?

Reply:
I added some tips at the top of AnalyticsEvents.js. Let me know if that's not clear, or if the place is not right